### PR TITLE
alter schema start time userscdr billablecalls

### PR DESCRIPTION
**Pull request by danigargar**
#### Type Of Change 
- [x] Small bug fix
- [ ] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, portal/platform, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

File: library/phpstan-baseline.neon [hunk 1]
```diff
@@ -3725,6 +3730,11 @@ parameters:
 			count: 1
 			path: Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdr.php
 
+		-
+			message: "#^Parameter \\#1 \\$startTime of method Ivoz\\\\Provider\\\\Domain\\\\Model\\\\BillableCall\\\\BillableCallDtoAbstract\\:\\:setStartTime\\(\\) expects DateTimeInterface\\|string, DateTimeInterface\\|string\\|null given\\.$#"
+			count: 1
+			path: Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdr.php
+
```
Comment (rbatistadev) [inline library/phpstan-baseline.neon:3737]: I'm pretty sure you can fix this errors without ignoring them

File: library/psalm-baseline.xml [hunk 2]
```diff
@@ -4501,15 +4501,19 @@
     </MixedArrayAssignment>
   </file>
   <file src="Ivoz/Provider/Domain/Service/BillableCall/CreateOrUpdateByTrunksCdr.php">
-    <PossiblyNullArgument occurrences="1">
+    <PossiblyNullArgument occurrences="2">
       <code>$trunksCdrDto-&gt;getDuration()</code>
+      <code>$trunksCdrDto-&gt;getStartTime()</code>
```
Comment (rbatistadev) [inline library/psalm-baseline.xml:4506]: same here

File: library/psalm-baseline.xml [hunk 3]
```diff
@@ -4501,15 +4501,19 @@
...
     </PossiblyNullArgument>
   </file>
   <file src="Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdr.php">
     <MixedArgument occurrences="2">
       <code>$destinationName</code>
       <code>$ratingPlanGroupName</code>
     </MixedArgument>
+    <PossiblyNullArgument occurrences="1">
+      <code>$startTime</code>
+    </PossiblyNullArgument>
```
Comment (rbatistadev) [inline library/psalm-baseline.xml:4516]: and here

File: library/spec/Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdrSpec.php [hunk 4]
```diff
@@ -146,6 +146,10 @@ private function prepareExecution()
             BillableCallDto::class
         );
 
+//        $this->billableCallDto
+//            ->getStartTime()
+//            ->willReturn(new \DateTime('2000-01-01 00:00:00'));
```
[the remark this answers is not in the document]
Comment (danigargar) [reply library/spec/Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdrSpec.php:151]: Ups, thank you!

File: library/psalm-baseline.xml (hunk 2 above)
Comment (danigargar) [reply library/psalm-baseline.xml:4506]: My previous comment also applies here, plus getDuration (a really similar situacion), was already there an marks a precedent.

File: library/phpstan-baseline.neon (hunk 1 above)
Comment (danigargar) [reply library/phpstan-baseline.neon:3737]: Not that easy: I should modify generated code (like BillableCallDtoAbstract.php) or ignore it with a comment in code not directly related to this PR:
https://github.com/irontec/ivozprovider/blob/2bd8decc33d782ce69edf94c6ffb06365c346bbf/library/Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdr.php#L46-L49

**Comment by rbatistadev** [comment]
Checkout my last commit please

**Review by rbatistadev — APPROVED** [review]

**Pull request merged**